azext_iot/tests/digitaltwins/__init__.py

```diff
@@ -166,7 +166,7 @@ def wait_for_hostname(
         sleep(wait_in_sec)
 
         self.embedded_cli.invoke(
-            "dt wait -n {} -g {} --custom \"hostName && provisioningState=='Succeeded'\" --interval {} --timeout {}".format(
+            "dt wait -n {} -g {} --created --interval {} --timeout {}".format(
```

Comment:
Just to jog my memory - can you explain the difference here? Does `--created` return when the poller does (possibly still `provisioning`) or does it wait for our create method to finish?

Reply:
wait --created checks if the get call returns a provisioningState that is not Provisioning. Thus, this will depend on the digital twins get calls and if that returned has a provisioningState of completed. This is assuming that the get call will only return a digital twins object that has provisioningState == Succeeded if there is a hostname.